src/Audit/CustomDomains.php

```diff
@@ -10,23 +10,59 @@
  */
 class CustomDomains extends Audit {
 
-  /**
-   * @inheritdoc
-   */
-  public function audit(Sandbox $sandbox) {
+	/**
+	 * @inheritdoc
+	 */
+	public function audit(Sandbox $sandbox) {
+		$warn = FALSE;
 
-    $domains = array_filter($this->target['acquia.cloud.environment.domains'], function ($domain) {
-      // Do not include ELB domains or Acquia default domains.
-      return !(strpos($domain, 'acquia-sites.com') || strpos($domain, 'elb.amazonaws.com') || strpos($domain, 'acsitefactory.com'));
-    });
+		foreach ($this->target['acquia.cloud.environment.domains'] as $domain) {
+			if ($this->urlResponds($domain) == FALSE) {
```

Comment:
Wow this is going to add some significant performance lag to apps with many domains. 

1. We shouldn't have added performance cost to an existing audit.
2. We should be using curl multi to run requests in parallel
3. Have we thought about the potential for performance related issues by making so many requests to large multisite instances?